pkg/oc/bootstrap/docker/openshift/helper.go

```diff
@@ -848,7 +848,6 @@ func (h *Helper) updateConfig(configDir string, opt *StartOptions) error {
 window.OPENSHIFT_CONSTANTS.ENABLE_TECH_PREVIEW_FEATURE = {
   service_catalog_landing_page: true,
   template_service_broker: true,
```

Comment:
Trailing comma here

Reply:
yeah i confused myself into thinking that was go code when the tests were failing last night. sigh. fixe.d